src/universal/components/AnalyticsIdentifier.tsx

```diff
@@ -37,7 +37,7 @@ class AnalyticsIdentifier extends Component<Props, State> {
       avatar,
       created,
       email,
-      name
+      parabolPreferredName: name
```

Comment:
i think your comment was meant for this one?
if so, i can't remember why we did this. maybe before all the ID calls on the server?
Seems safe enough to remove :man_shrugging:

Reply:
Oh yes, correct. The comment was supposed to be for this one.

Here's my guess: this identify() call is the only place to send the anonymous segment id to later get alias()ed on the server. The only issue I noticed with this call is that it gets called for everybody quite often. It's generating extra events.

Any objection to me adding a // TODO: above this to look at it and consider limiting it to only sending when there is anonymous id to share with segment?

Reply:
nah, the segment id gets sent along with a signup, this never gets called for anonymous folks.
i'd be fine just removing the call to window.anlytics.identify & seeing what mayhem it causes.

Reply:
Ok, I agree. Let's pull it and see what breaks.